src/node_quic_crypto.cc

```diff
@@ -0,0 +1,1827 @@
+#include "node_quic_crypto.h"
```

Comment:
Review note: previously this was all inlined functions in node_quic_crypto.h but that really never made sense (well, it did very early on but stopped making sense a while ago). Unfortunately, quite a few of the functions have changed but the move here makes that harder to see.